plugin.js

```diff
@@ -56,16 +56,22 @@ function rawBody (fastify, opts, next) {
   next()
 
   function preparsingRawBody (request, reply, payload, done) {
+    const applyLimit = request.context._parserOptions.limit || fastify.initialConfig.bodyLimit
```

Comment:
should you check for null or undefined instead? is 0 technically a valid limit?

Reply:
Fastify rejects it

```
'bodyLimit' option must be an integer > 0. Got '0'
```

I fixed the check anyway by using the `??` operator instead